app/serializers/api/variant_serializer.rb

```diff
@@ -41,10 +41,16 @@ def thumb_url
   end
 
   def unit_price_price
-    (rand * 10).round(2)
+    price_with_fees / unit_price.denominator
   end
 
   def unit_price_unit
-    rand > 0.5 ? "item" : "kg"
+    unit_price.unit
+  end
+
+  private
+
+  def unit_price
+    @unit_price ||= UnitPrice.new(object)
```

Comment:
The build is failing, with errors pointing to this line, for ex.:
```
LineItemsController listing bought items lists items bought by the user from the same shop in the same order_cycle
     Failure/Error: @unit_price ||= UnitPrice.new(object)
```

Reply:
```diff
-- UnitPrice.new
++ ::UnitPrice.new
```
Might need a rebase as well, we've merged a lot in last 9 days... :+1:

Reply:
@filipefurtad0 thanks! I missed that. It was actually a problem with the file name Matt - `unit_prices.rb` was defining the `UnitPrice` class. I renamed it and it looks like it should be fixed.

Reply:
Thanks @andrewpbrett ,
Looking at the build (both on semaphore and github), it looks like job nr.3 did not run, due to this error:
`An error occurred while loading ./spec/services/unit_prices_spec.rb`

A question from my side: to check the sanity of the build, should I consider semaphore, github or both? :-)

Reply:
Sorry about that @filipefurtad0 - should have waited for a green build before notifying you. 

I made a small change (I'd missed a place where we refer to UnitPrices, plural) and now that spec passes locally for me. 

I'd say we're at a point where both builds *should* be green before starting testing. If not green, green-ish: there are still one or two flaky specs that I'm seeing in both builds sometimes. And it may take a while before all PR's catch up if they branched off master a while ago.